workflow/kfdrc_production_WES_somatic_variant_cnv_wf.cwl

```diff
@@ -109,6 +109,16 @@ outputs:
   lancet_prepass_vcf: {type: File, outputSource: run_lancet/lancet_prepass_vcf}
 
 steps:
+  bedtools_intersect_germline:
+    run: ../tools/bedtools_intersect.cwl
+    in:
+      input_vcf: b_allele
+      output_basename: output_basename
+      input_bed_file: unpadded_capture_regions
+      flag: i_flag
```

Comment:
Within the script, my doc is totally wrong. I think I had copied and modified from another, doc should be something like: `If WXS, filter germline on unpadded intervals to ensure accurate CNV calls`

Reply:
I will address in another PR, it's a small issue and shouldn't hold this one back